net-libs/gloox/gloox-1.0.28.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+MY_P="${P/_/-}"
+DESCRIPTION="A portable high-level Jabber/XMPP client library for C++"
+HOMEPAGE="https://camaya.net/gloox/"
+SRC_URI="https://camaya.net/download/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-3"
+# Check upstream changelog: https://camaya.net/gloox/changelog/
+# REVIEW(fkobi): I see no mention of 18 there and no change
+# from 1.0.24 to 1.0.28 in https://camaya.net/api/gloox-1.0.2[48]/
+SLOT="0/18"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="debug gnutls idn ssl static-libs test zlib"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	idn? ( net-dns/libidn:= )
+	gnutls? ( net-libs/gnutls:= )
+	ssl? ( dev-libs/openssl:0= )
+	zlib? ( sys-libs/zlib )
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		--without-examples # not installed anyway so don't build them
```

Comment:
Why aren't they installed? :) Maybe you want to build then with `+examples` use flag?